server/tests/models/group.test.js

```diff
@@ -0,0 +1,99 @@
+import { expect } from 'chai';
+import Group from '../../models/group';
+import EscalationPolicy from '../../models/escalationPolicy';
+
+describe('Groups', () => {
+  const subscriberRotationInterval = 7;
+  const subscriberPagingInterval = 15;
+  const subscriberObjectId = '57e590a0140ebf1cc48bb1bf';
+
+  const userObjectId = '67e590a0140ebf1cc48bb1bf';
+
+  context('with a valid group', () => {
+    const escPolicy = {
+      rotationInterval: subscriberRotationInterval,
+      pagingInterval: subscriberPagingInterval,
+      subscribers: [subscriberObjectId]
+    }
+
+    const groupData = {
+      users: [userObjectId],
+      escalationPolicy: escPolicy
+    }
+
+    const group = new Group(groupData);
+
+    it('should create a new group', (done) => {
+      group.save((err, group) => {
+        if(err) {
+          done(err);
+        }
+        expect(group.users).to.be.a('array');
+        expect(group.escalationPolicy.subscribers).to.be.a('array');
+        done();
+      });
+    });
+  });
+
+  context('when rotation interval is not there', () => {
+    const escPolicy = {
+      rotationInterval: null,
+      pagingInterval: subscriberPagingInterval,
+      subscribers: [subscriberObjectId]
+    }
+
+    const groupData = {
+      users: [userObjectId],
+      escalationPolicy: escPolicy
+    }
+
+    const group = new Group(groupData);
+
+    it('should create a new group', (done) => {
+      group.save((err, group) => {
+        if(err) {
```

Comment:
I like where you're going here. There are two approaches that I would just clean this up. According to [Mocha](mochajs.org), you can do this in one line,
`if (err) done(err);` 

However, in other tests, we've just been asserting that the error doesn't exist.
`expect(err).to.not.exist;`

Personally, I like chai doing the assertions, however, the asertion doesn't give you the reason why it failed.